README.rdoc

```diff
@@ -54,12 +54,19 @@ The configuration options currently supported are:
   This parameter is *required* unless the +auto_create_table+ option is used.
   
 +endpoint+::
```

Comment:
Do you need to change this parameter to `import_endpoint` since you changed [here](https://github.com/treasure-data/fluent-plugin-td/pull/31/files#diff-ead51eefe64721e09611ee844421ab52L25)?